test/other/codesize/hello_libcxx.cpp

```diff
@@ -5,8 +5,7 @@
 
 #include <iostream>
 
-int main()
-{
+int main() {
```

Comment:
We have clang-format config, is it not applied to tests or not enforced in general?

Could be nice if it was enforced the same way prettier is.